test/Constraints/pack-expansion-expressions.swift

```diff
@@ -77,3 +77,19 @@ func sameShapeDiagnostics<T..., U...>(t: repeat each T, u: repeat each U) {
   _ = repeat Array<(each T, each U)>() // expected-error {{pack expansion requires that 'U' and 'T' have the same shape}}
   _ = repeat (Array<each T>(), each u) // expected-error {{pack expansion requires that 'U' and 'T' have the same shape}}
 }
+
+func returnPackExpansionType<T...>(_ t: repeat each T) -> repeat each T { // expected-error {{variadic expansion 'T' cannot appear outside of a function parameter list, function result, tuple element or generic argument list}}
```

Comment:
This error message seems inaccurate. Is says an expansion cannot appear outside a function result, but syntactically the expansion *is* the function result.

Reply:
I think "function result" should just be removed from the message. A previous experimental implementation of variadic generics that modeled packs as tuples (where this message was added) allowed returning a pack type `T`, but the current design does not allow for that. I also think we should eliminate the phrase "variadic expansion". I can update this message in a separate PR.